geochronometria.csl

```diff
@@ -81,7 +81,7 @@
         </group>
       </if>
       <else>
-        <text variable="genre" suffix=". "/>
+        <text variable="genre"/>
         <group delimiter=": ">
           <text variable="publisher"/>
           <text variable="publisher-place"/>
```

Comment:
@dbobak, did you actually test this change? It looks like this changes "genre. publisher: publisher-place" to "genrepublisher: publisher-place", which seems incorrect.